browser/net/brave_network_delegate_browsertest.cc

```diff
@@ -204,36 +204,36 @@ class BraveNetworkDelegateBrowserTest : public InProcessBrowserTest {
   GURL wp_top_url_;
   GURL wp_frame_url_;
   GURL a_frame_url_;
+  net::test_server::EmbeddedTestServer https_server_;
 
  private:
   ContentSettingsPattern top_level_page_pattern_;
   ContentSettingsPattern first_party_pattern_;
   ContentSettingsPattern iframe_pattern_;
-  net::test_server::EmbeddedTestServer https_server_;
 };
 
 // It is important that cookies in following tests are set by response headers,
 // not by javascript. Fetching such cookies is controlled by NetworkDelegate.
 IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest, Iframe3PCookieBlocked) {
   ui_test_utils::NavigateToURL(browser(), url_);
-  const std::string cookie =
-      content::GetCookies(browser()->profile(), GURL("http://c.com/"));
+  const std::string cookie = content::GetCookies(
+      browser()->profile(), https_server_.GetURL("c.com", "/"));
   EXPECT_TRUE(cookie.empty()) << "Actual cookie: " << cookie;
 }
 
 IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest, Iframe3PCookieAllowed) {
   AllowCookies(top_level_page_url_);
   ui_test_utils::NavigateToURL(browser(), url_);
-  const std::string cookie =
-      content::GetCookies(browser()->profile(), GURL("http://c.com/"));
+  const std::string cookie = content::GetCookies(
+      browser()->profile(), https_server_.GetURL("c.com", "/"));
   EXPECT_FALSE(cookie.empty());
 }
 
 IN_PROC_BROWSER_TEST_F(BraveNetworkDelegateBrowserTest, Iframe3PShieldsDown) {
   ShieldsDown(top_level_page_url_);
   ui_test_utils::NavigateToURL(browser(), url_);
   const std::string cookie =
-      content::GetCookies(browser()->profile(), GURL("http://c.com/"));
+      content::GetCookies(browser()->profile(), GURL("https://c.com/"));
```

Comment:
curious, why above we use `https_server_` go get the url and here we don't? @bridiver

Reply:
good question, I was just doing a search/replace, but since port doesn't matter it should be fine